templates/BUILD.tpl

```diff
@@ -210,17 +210,18 @@ cc_toolchain_config(
     abi_version = "local",
     abi_libc_version = "local",
     cxx_builtin_include_directories = [
+        # Keep this list in sync with "templates/include_directory_paths_gcc"
```

Comment:
Instead of duplicating paths here and in Starlark, could we move the fixed lists of include directories from the BUILD file template to a `.bzl` file?

Reply:
Good call. That's cleaner.